behat_suites.yml

```diff
@@ -1,15 +1,13 @@
 # This file is meant to be imported from ezplatform's behat.yml.dist.
 # All path are relative to the root ezplatform directory.
-default:
-  suites:
-    adminui:
-      paths:
-        - %paths.base%/vendor/ezsystems/ezplatform-admin-ui/src/features
-      filters:
-        tags: common
-      contexts:
-        - EzSystems\EzPlatformAdminUi\Behat\UtilityContext
-        - EzSystems\EzPlatformAdminUi\Behat\AuthenticationContext
-        - EzSystems\EzPlatformAdminUi\Behat\NavigationContext
-
-  
+adminui:
+    suites:
+        adminui:
+            paths:
+                - '%paths.base%/vendor/ezsystems/ezplatform-admin-ui/src/features'
+            filters:
+                tags: "@common && ~@broken"
```

Comment:
Not a fan of this change, at this moment there are no "broken" tests and I'd like to keep it that way :P

Reply:
> Not a fan of this change, at this moment there are no "broken" tests and I'd like to keep it that way :P

Sorry, I forgot eZ Platform Admin UI tests are all amazing :grinning: Removed via c13b94a,